.github/workflows/backend.yml

```diff
@@ -384,6 +409,12 @@ jobs:
           # https://github.com/codecov/codecov-bash/blob/7100762afbc822b91806a6574658129fe0d23a7d/codecov#L891
           fetch-depth: '2'
 
+      - name: Update environment for silo databases
+        if: |
+          contains( github.event.pull_request.labels.*.name, 'Trigger: Silo db' )
+        run: |
+          echo '::set-env name=SENTRY_USE_SILOS=1'
```

Comment:
these `set-env` commands haven't worked for months -- you're looking for `$GITHUB_ENV` https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable